capstone/src/main/webapp/explore-loader.js

```diff
@@ -87,7 +94,7 @@ function editButton(club, joinedClubList, className) {
           ? function() {sendJoinedAlert('interested', true, club)} 
           : function() {sendJoinedAlert('joined', true, club)};
     
-      if (button.innerText == 'Join Club') {
+      if (button.innerText == 'Join Club' || button.innerText == 'Request to Join') {
         button.innerText = 'Joined';
```

Comment:
Maybe this is a future thing but would it be better to have it separated into 2 cases? One where the inner text is 'Joined' and the other 'Requested'? Given that a request can be rejected so they haven't officially joined yet?

Reply:
Yeah, that makes sense that joined and requested should be handled separately, but we probably wouldn't have time to do it. Thanks for bringing it up!